packages/react-storage/src/components/StorageBrowser/views/Controls/Paginate.tsx

```diff
@@ -48,17 +42,25 @@ const getButtonVariantProps = (
     case 'paginate-next':
       ariaLabel = 'Go to next page';
       className = `${BLOCK_NAME}__button-next`;
-      disabled = disableNext;
-      onClick = handleNext;
+      disabled =
+        !!currentPage &&
+        !!highestPageVisited &&
```

Comment:
Is page 0-indexed? can `currentPage` or `highestPageVisited` be 0?

Reply:
`currentPage` is 1 indexed and will always be above 0. I would imagine `highestPageVisited` can be zero, since it's related to returned results. However, my understanding is that we have to include these checks because the paginate control props cannot be required.